python/tests/test_nearest_neighbors.py

```diff
@@ -357,6 +363,9 @@ def test_lsh_spark_compat(gpu_number: int) -> None:
     from pyspark.ml.linalg import Vectors
     from pyspark.sql.functions import col
 
+    # reduce the number of GPUs for toy dataset to avoid empty partition.
+    # cuml backend requires k <= the number of rows in the smallest index partition.
+    gpu_number = min(gpu_number, 1)
```

Comment:
same here.

Reply:
Revised to use 2 GPU and increased the size of the toy dataset.